isis/appdata/import/PDS3/MessMdis.tpl

```diff
@@ -0,0 +1,88 @@
+{% extends "img_base.tpl" %}
+
+{% block instrument %}
+    SpacecraftName               = Messenger
+    InstrumentName               = "{{INSTRUMENT_NAME.Value}}"
+    InstrumentId                 = {{INSTRUMENT_ID.Value}}
+    TargetName                   = "{{TARGET_NAME.Value}}"
+    OriginalTargetName           = "{{TARGET_NAME.Value}}"
+    StartTime                    = {{START_TIME.Value}}
+    StopTime                     = {{STOP_TIME.Value}}
+    SpacecraftClockCount         = {{SPACECRAFT_CLOCK_START_COUNT.Value}}
+    MissionPhaseName             = "{{MISSION_PHASE_NAME.Value}}"
+    ExposureDuration             = "{{MESS_EXPOSURE.Value}} <MS>"
+    ExposureType                 = {{EXPOSURE_TYPE.Value}}
+    DetectorTemperature          = {{DETECTOR_TEMPERATURE.Value}} <DEGC>
+    FocalPlaneTemperature        = {{FOCAL_PLANE_TEMPERATURE.Value}} <DEGC>
+    FilterTemperature            = {{FILTER_TEMPERATURE.Value}}
+    OpticsTemperature            = {{OPTICS_TEMPERATURE.Value}} <DEGC>
+    {% set ok_levels = ["7","6","5"] %}
+    {% set bad_levels = ["3","2","1","0"] %}
+    AttitudeQuality              = {% if MESS_ATT_FLAG.Value in ok_levels %}
+                                    "Ok" 
+                                    {% else if MESS_ATT_FLAG.Value == 4 %}
+                                    "Illegal"
+                                    {% else if MESS_ATT_FLAG.Value in bad_levels %}
+                                    "Bad"
+                                    {% else %}
+                                    "UNKNOWN"
+                                    {% endif %}
+    FilterWheelPosition          = {{MESS_FW_POS.Value}}
+    PivotPosition                = {{MESS_PIV_POS.Value}}
+    FpuBinningMode               = {{MESS_FPU_BIN.Value}}
+    PixelBinningMode             = {{MESS_PIXELBIN.Value}}
+    SubFrameMode                 = {{MESS_SUBFRAME.Value}}
+    JailBars                     = {{MESS_JAILBARS.Value}}
+    DpuId                        = {{MESS_DPU_ID.Value}}
+{% endblock %}
```

Comment:
Unlutted and PivotAngle are not here, the former comes from an input parameter is seems and the latter is computed, any suggestions on this?

Reply:
The input parameter poses one of our problems. Make sure it is noted in the mdis section of the word doc. Do you have access to the values to compute the other? If so go ahead and do the math in the template and add it. If not note it for in word and we'll add it to the list of other difficulties.

Reply:
I noted it in the doc.